package.json

```diff
@@ -6,7 +6,7 @@
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "engineStrict": "true",
   "engines": {
-    "node": ">=0.10",
+    "node": ">=4.0",
```

Comment:
I think this is the right spot? I agree that there should be a sockit-to-me version bump, and I'm fine tracking against node versions if we want to.